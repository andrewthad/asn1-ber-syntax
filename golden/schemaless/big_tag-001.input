5FA73002 8109
