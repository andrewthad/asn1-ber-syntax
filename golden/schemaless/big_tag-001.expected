Value
    { tagClass = Application
    , tagNumber = 5040
    , contents = Primitive
        [ 0x81
        , 0x09
        ]
    }
